SSOT/UpdateOccupationsWithNOC2021.sql

```diff
@@ -7,7 +7,7 @@ FROM OPENJSON (@JSON)
 WITH 
 (
     [noc_2021] varchar(10), 
-    [label] varchar(50) 
+    [label] varchar(500) 
```

Comment:
This needs to be the exact same type as the destination column.

Reply:
Done the change